Equilobe.DailyReport.Models/Storage/ReportSettings.cs

```diff
@@ -26,7 +27,7 @@ public class ReportSettings
 
         public string PolicyXml { get; set; }
 
-        // NO DB
-        //public Policy Policy { get; set; }
+        [NotMapped]
+        public JiraPolicy Policy { get; set; }
```

Comment:
@vladichimescu 
Nu inteleg de ce ai mai decomentat proprietatea asta avand in vedere ca nu faci NIMIC cu ea. 

Proprietatea asta avea sensul sa faca automat serializarea/deseraializarea pe get/set, dar tu nu ai folosit-o de maniera asta, deci nu prea are sens momentan.